box.json.dist

```diff
@@ -0,0 +1,20 @@
+{
+    "check-requirements": false,
+    "chmod": "0755",
+    "compactors": [
+        "Herrera\\Box\\Compactor\\Php"
+    ],
+    "files": [
```

Comment:
question:
`finder` key is no longer available? :(

Reply:
it is, but why bother with a complicated manual `finder` config when you can have Box figuring out for you?

Box tries to pick the relevant file by checking the `composer.json` file. There is however sometimes some file which are not managed by Composer like the two ones listed.

Reply:
I'm simply trying to figure out size difference for phar built by box2 and box3